app/main/validate-config.ts

```diff
@@ -0,0 +1,23 @@
+import fs from 'fs';
+import path from 'path';
+import { App } from 'electron';
+
+export function validateConfig(app: App) {
+  const configPath = path.join(app.getPath('userData'), 'config.json');
+  fs.readFile(configPath, 'utf8', (err, walletConfigText) => {
+    if (err) return;
+    try {
+      JSON.parse(walletConfigText);
```

Comment:
Should we validate that the correct keys are in here? It seems like the config could still be valid JSON, but not have the keys that we need. If the wallet will still work fine in that case, then this is OK.

Reply:
Yeah that is a good idea, though I think I'll put that logic elsewhere though opening issue https://github.com/blockstack/stacks-wallet/issues/481